### ISSUE TITLE
Adding icons to internal links causes it to wrap or not align correctly

### ISSUE THREAD

**Issue by tww-mastery** [chrisgurney/obsidian-note-toolbar#117]
**Description of the bug**
When I add text, including emoji characters, or an Iconize icon to a note callout item where it links to a note rather than a URL or command, the item wraps and occupies two lines.

I see this even in the Sandbox vault with only Note Toolbar enabled.

**Steps to reproduce**

Create a note callout with an icon:

```md
> [!note-toolbar]
> - [Some Website](https://google.com)
> - [:LiPin: Some Command](obsidian://...)
> - ✅  [[Some Note]]
```

Notice that the  item with the icon wraps and occupies two lines. I notice the same behavior with Emoji characters or normal text or anything outside the double brackets of the note link. There is plenty of space in the note for it to expand to the right.

This also happens regardless of position within the toolbar (it doesn't only have to be at the end).

**Expected behavior**

The contents stay on one line.

**Screenshots (if you think it will help)**
![image](https://github.com/user-attachments/assets/f0eca5e1-43b4-46d2-81e3-6ee65def2939)

**Debug info:**
```
SYSTEM INFO:
	Obsidian version: v1.7.0
	Installer version: v1.6.5
	Operating system: Darwin Kernel Version 22.6.0: Mon Apr 22 20:54:28 PDT 2024; root:xnu-8796.141.3.705.2~1/RELEASE_X86_64 22.6.0
	Login status: logged in
	Catalyst license: vip
	Insider build toggle: on
	Live preview: on
	Base theme: adapt to system
	Community theme: none
	Snippets enabled: 0
	Restricted mode: off
	Plugins installed: 1
	Plugins enabled: 1
		1: Note Toolbar v1.10.2

RECOMMENDATIONS:
	Community plugins: for bugs, please first try updating all your plugins to latest. If still not fixed, please try to make the issue happen in the Sandbox Vault or disable community plugins.
```

**Plugin, Theme links, and CSS snippets (if applicable):**
_Plugins, themes, and CSS snippets can affect the display of the toolbar. Include any being used here to help diagnose display issues._

**Additional context**
_Add any other context about the problem here._

**Comment by chrisgurney** [comment]
Thanks for the bug report @tww-mastery!

Try putting the emoji inside the link instead:
```markdown
> - [[✅ Some Note]]
```

**Comment by chrisgurney** [comment]
@tww-mastery OK I've got a fix in 1.10.3, and this is the best I can do, for now:

![CleanShot 2024-08-22 at 15 57 24@2x](https://github.com/user-attachments/assets/bc28d293-d065-4dde-99e8-a127efd9b741)

CSS doesn't let me target these elements specifically, though I'm open to a front-end developer taking a look!

**Comment by tww-mastery** [comment]
Thank you very much!
